basic-javascript/tests/project-2.test.js

```diff
@@ -1,4 +1,334 @@
 const assert = require('chai').assert;
 const funcs = require('../src/project-2');
 
-// whoops.. there is no test suite for this file. You'll simply just have to create one :/
+describe('Project Two', () => {
+    let mockArray = []
```

Comment:
Missing semicolons.

Also, try to stay consistent with identations (if you use 4 spaces in one file, maintain consistency in all files)